practiseClient/script.js

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
The separation of folders is confusing, would recommend moving script.js into the client folder and removing any files/code not in use. Adding some comments would also be useful to help with understanding the code.